docs/api-docs/analytics/bodl-overview.md

```diff
@@ -0,0 +1,423 @@
+
+    sampleAnalyticsProvider.load('<%= property_id %>');
+    sampleAnalyticsProvider.page();
+
+    // Advanced Matching
+    if (BODL.customer && BODL.customer.id) {
+      var customerObj = {
+        email: BODL.customer.email,
+      }
+
+      if (BODL.customer.phone) {
+        var phoneNumber = BODL.customer.phone;
+        if (BODL.customer.phone.indexOf('+') === -1) {
+          // No country code, so default to US code
+          phoneNumber = `+1${phoneNumber}`;  
+        }
+
+        customerObj.phone = phoneNumber;
+      }
+
+      sampleAnalyticsProvider.identify(BODL.customer.id, customerObj);
+    }
+  }(window, document, 'sampleAnalyticsProvider');
+</script>
+```
+
+### Add to cart
+
+The following snippet handles the Add to Cart event by collecting data on each item added, including the product ID, the quantity added, and the price.
+
+```handlebars title="Sample Pixel Code Start: Product Detail Page Add to Cart Event" lineNumbers
+<script>
+  document.querySelectorAll('[data-cart-item-add]').forEach(form => form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    let productId, productQty;
+    for (const pair of formData.entries()) {
+      if (pair[0] === 'product_id') {
+        productId = pair[1];
+      } else if (pair[0] === 'qty[]') {
+        productQty = pair[1];
+      }
+    }
+
+    // References:
+    // https://developer.bigcommerce.com/theme-objects/product
+    // https://developer.bigcommerce.com/stencil-docs/developing-further/catalog-price-object
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('AddToCart', {
+      content_id: BODL.product.id,
+      content_category: BODL.breadcrumbs[1] ? BODL.breadcrumbs[1].name : '',
+      content_type: 'product_group',
+      content_name: BODL.product.title,
+      quantity: productQty,
+      price: BODL.product.price.without_tax.value,
+      value: (BODL.product.price.without_tax.value * productQty),
+      currency: BODL.product.price.without_tax.currency,
+    });
+    
+  }}));
+
+  if (BODL.cartItemAdded) {
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('AddToCart', {
+      content_id: BODL.cartItemAdded.product_id,
+      content_type: 'product_group',
+      content_name: BODL.cartItemAdded.name,
+      quantity: BODL.cartItemAdded.quantity,
+      price: BODL.cartItemAdded.price.value,
+      value: BODL.cartItemAdded.total.value,
+      currency: BODL.cartItemAdded.price.currency,
+    });
+  }
+</script>
+```
+
+### Add to wishlist
+The following snippet handles the Add to Wishlist event. This snippet sends individual products specified by the `added_product_id` and a commented section regarding category data. Our built-in analytics do not currently track wishlists' category data, but the snippet includes it for future-proofing.
```

Comment:
@becomevocal , I think the point around "Our built-in analytics do not currently track wishlists' category data, but the snippet includes it for future-proofing." can be removed. Whether or not our built-in control panel analytics does anything with wishlist, to me, doesn't matter. If the BODL supports it, then it can be of use to tracking scripts. What's your take Nate?

Reply:
Yep! Make sense to remove since it's up to the developer to utilize or not.